server/base/src/test/java/org/apache/accumulo/server/conf/AccumuloConfigurationIsPropertySetTest.java

```diff
@@ -0,0 +1,327 @@
+    SystemPropKey sysPropKey = SystemPropKey.of(instanceId);
+    VersionedProperties sysProps = new VersionedProperties(1, Instant.now(),
+        Map.of(GC_PORT.getKey(), "1234", TSERV_SCAN_MAX_OPENFILES.getKey(), "19"));
+    expect(propStore.get(eq(sysPropKey))).andReturn(sysProps).times(2);
+    replay(propStore);
+    ConfigurationCopy defaultConfig = new ConfigurationCopy(
+        Map.of(TABLE_BLOOM_SIZE.getKey(), TABLE_BLOOM_SIZE.getDefaultValue()));
+
+    Set<Property> shouldBeSet = Set.of(TABLE_BLOOM_SIZE, GC_PORT, TSERV_SCAN_MAX_OPENFILES);
+    Set<Property> shouldNotBeSet = Sets.difference(shouldBeSet, ALL_PROPERTIES);
+
+    // create SystemConfiguration object
+    SystemConfiguration systemConfiguration =
+        new SystemConfiguration(context, sysPropKey, defaultConfig);
+
+    verifyProps(systemConfiguration, shouldBeSet, shouldNotBeSet);
+
+    testPropertyIsSetImpl(systemConfiguration, shouldBeSet, shouldNotBeSet);
+  }
+
+  @Test
+  public void tableConfiguration() {
+    InstanceId instanceId = InstanceId.of(UUID.randomUUID());
+    ServerContext context = createMock(ServerContext.class);
+    expect(context.getInstanceID()).andReturn(instanceId).anyTimes();
+    PropStore propStore = createMock(PropStore.class);
+    expect(context.getPropStore()).andReturn(propStore).anyTimes();
+    var siteConfig = SiteConfiguration.empty().build();
+    expect(context.getSiteConfiguration()).andReturn(siteConfig).anyTimes();
+    replay(context); // prop store is read from context.
+    propStore.registerAsListener(anyObject(), anyObject());
+    expectLastCall().anyTimes();
+    var sysPropKey = SystemPropKey.of(instanceId);
+    VersionedProperties sysProps =
+        new VersionedProperties(1, Instant.now(), Map.of(TABLE_BLOOM_ENABLED.getKey(), "true"));
+    expect(propStore.get(eq(sysPropKey))).andReturn(sysProps).times(2);
+    NamespaceId NID = NamespaceId.of("namespace");
+    var nsPropKey = NamespacePropKey.of(instanceId, NID);
+    VersionedProperties nsProps = new VersionedProperties(2, Instant.now(),
+        Map.of(TABLE_FILE_MAX.getKey(), "21", TABLE_BLOOM_ENABLED.getKey(), "false"));
+    expect(propStore.get(eq(nsPropKey))).andReturn(nsProps).once();
+    var TID = TableId.of("3");
+    var tablePropKey = TablePropKey.of(instanceId, TID);
+    VersionedProperties tableProps =
+        new VersionedProperties(3, Instant.now(), Map.of(TABLE_BLOOM_ENABLED.getKey(), "true"));
+    expect(propStore.get(eq(tablePropKey))).andReturn(tableProps).once();
+    ConfigurationCopy parentConfig =
+        new ConfigurationCopy(Map.of(TABLE_BLOOM_SIZE.getKey(), TABLE_BLOOM_SIZE.getDefaultValue(),
+            TABLE_DURABILITY.getKey(), TABLE_DURABILITY.getDefaultValue()));
+    replay(propStore);
+
+    NamespaceId nsid = nsPropKey.getId();
+
+    NamespaceConfiguration namespaceConfig =
+        new NamespaceConfiguration(context, nsid, parentConfig);
+
+    Set<Property> shouldBeSet = Set.of(TABLE_BLOOM_SIZE);
```

Comment:
Seems like this set should be the following based on what is set for the parent and table conf object.

```suggestion
    Set<Property> shouldBeSet = Set.of(TABLE_BLOOM_SIZE, TABLE_DURABILITY, TABLE_BLOOM_ENABLED);
```